README.md

```diff
@@ -0,0 +1,81 @@
+# Azure Functions Templates
+This repository contains the templates used by the [Azure Functions Portal](https://functions.azure.com/signin). Templates are pre-defined functions that demonstrate a working scenario and could be used as a starting point for more complex scenarios.
+
+## Template Format
+Template requires specific metadata files and folder structure so that [Azure Functions Portal](https://functions.azure.com/signin) can understand and graphically present it. Please find more information on individual files and their contents below.
+
+- **Binding.Json:** This is a metadata file for all the bindings and their possible configuration settings. It is common across all templates and is located [here](Bindings/bindings.json). It also contains metadata for binding related UI Elements and the corresponding text.
+
+- **Function.Json:** This file contains binding data specific to each template. It provides valid values for the possible settings on a binding.
+
+- **Code file:** Code file holds the actual code executed by the template. The name of the file depends on the Target language used by the template. For Javascript the name of the code file is `index.js`. For all the other languages the file name is `run`, followed by the file extension specific to the language.
```

Comment:
May want to switch the file names around since all languages (including JS) work with a `run.ext` file, but JS **also** works with `index.js`